src/js/components/agency/statusOfFunds/VisualizationSection.jsx

```diff
@@ -45,17 +48,32 @@ const VisualizationSection = ({
     results,
     selectedSubcomponent,
     fetchFederalAccounts
-}) => (
-    <div className="status-of-funds__visualization">
-        <h6>{level === 1 ? selectedSubcomponent?.name : agencyName} by <strong>{levels[level]}</strong> for FY {fy}</h6>
-        <div className="status-of-funds__controls">
-            <RoundedToggle toggle={toggle} onToggle={onToggle} label="View Outlays" />
+}) => {
+    const [open, setOpen] = useState(false);
+    return (
+        <div className="status-of-funds__visualization">
+            <h6>{level === 1 ? selectedSubcomponent?.name : agencyName} by <span className="status-of-funds__emphasis">{levels[level]}</span> for FY {fy}</h6>
+            <div className="status-of-funds__controls">
+                <RoundedToggle toggle={toggle} onToggle={onToggle} label="View Outlays" />
+                <div className="status-of-funds__line-div" />
+                <div className="status-of-funds__accordion">
+                    <Accordion setOpen={setOpen} closedIcon="chevron-down" openIcon="chevron-up" title="What is this?">What is this?</Accordion>
+                </div>
+            </div>
+            {open &&
+            <div className="status-of-funds__what-content">
+                <FontAwesomeIcon icon="info-circle" className="status-of-funds__info-icon" />
+                <p className="status-of-funds__what-heading">What is an outlay?</p>
```

Comment:
According to the MAC this 'outlay' wants to be italicized, so I guess it's bold and italicized

Reply:
fixed this